examples/local-sync/README.md

```diff
@@ -1,87 +1,3 @@
-arangodb create tls keyfile --cacert=${CERTDIR}/tls-ca.crt --cakey=${CERTDIR}/tls-ca.key --keyfile=${CERTDIR}/cluster1/tls.keyfile --host=${IP} --host=localhost
-arangodb create tls keyfile --cacert=${CERTDIR}/tls-ca.crt --cakey=${CERTDIR}/tls-ca.key --keyfile=${CERTDIR}/cluster2/tls.keyfile --host=${IP} --host=localhost
-
-# Create client authentication certificates
-arangodb create client-auth ca --cert=${CERTDIR}/client-auth-ca.crt --key=${CERTDIR}/client-auth-ca.key
-arangodb create client-auth keyfile --cacert=${CERTDIR}/client-auth-ca.crt --cakey=${CERTDIR}/client-auth-ca.key --keyfile=${CERTDIR}/client-auth-ca.keyfile
-
-# Create JWT secrets
-arangodb create jwt-secret --secret=${CERTDIR}/cluster1/syncmaster.jwtsecret
-arangodb create jwt-secret --secret=${CERTDIR}/cluster1/arangodb.jwtsecret
-arangodb create jwt-secret --secret=${CERTDIR}/cluster2/syncmaster.jwtsecret
-arangodb create jwt-secret --secret=${CERTDIR}/cluster2/arangodb.jwtsecret
-```
-
-## Step 2: Start first & second cluster
-
-```bash
-export DATADIR=<directories where to store database files>
-mkdir -p ${DATADIR}
-
-arangodb --starter.data-dir=/${DATADIR}/cluster1 \
-    --starter.sync \
-    --starter.local \
-    --auth.jwt-secret=${CERTDIR}/cluster1/arangodb.jwtsecret \
-    --sync.server.keyfile=${CERTDIR}/cluster1/tls.keyfile \
-    --sync.server.client-cafile=${CERTDIR}/client-auth-ca.crt \
-    --sync.master.jwt-secret=${CERTDIR}/cluster1/syncmaster.jwtsecret \
-    --starter.address=${IP}
-
-## In another terminal
-export DATADIR=<directories where to store database files>
-
-arangodb --starter.data-dir=/${DATADIR}/cluster2 \
-    --starter.sync \
-    --starter.local \
-    --auth.jwt-secret=${CERTDIR}/cluster2/arangodb.jwtsecret \
-    --sync.server.keyfile=${CERTDIR}/cluster2/tls.keyfile \
-    --sync.server.client-cafile=${CERTDIR}/client-auth-ca.crt \
-    --sync.master.jwt-secret=${CERTDIR}/cluster2/syncmaster.jwtsecret \
-    --starter.address=${IP} \
-    --starter.port=9528
-```
-
-Note that it is not uncommon for a syncmaster to restart, since the cluster is not yet ready when it is started.
-
-## Step 3: Configure synchronization from cluster 1 to cluster 2
-
-```bash
-arangosync configure sync \
-    --master.endpoint=https://${IP}:9542 \
-    --master.keyfile=${CERTDIR}/client-auth-ca.keyfile \
-    --source.endpoint=https://${IP}:8542 \
-    --source.cacert=${CERTDIR}/tls-ca.crt \
-    --auth.keyfile=${CERTDIR}/client-auth-ca.keyfile
-```
-
-## Step 4: Check status of configuration
-
-```bash
-# Check status of cluster 1
-arangosync get status \
-    --master.endpoint=https://${IP}:8542 \
-    --auth.keyfile=${CERTDIR}/client-auth-ca.keyfile \
-    --verbose
-
-# Check status of cluster 2
-arangosync get status \
-    --master.endpoint=https://${IP}:9542 \
-    --auth.keyfile=${CERTDIR}/client-auth-ca.keyfile \
-    --verbose
-```
+This page has been moved [here](../../docs/Manual/Tutorials/Starter/LocalDC2DC.md)
```

Comment:
"here" link labels should be avoided
```suggestion
The content has been moved, see [Local DC2DC Starter Tutorial](../../docs/Manual/Tutorials/Starter/LocalDC2DC.md).
```

I wonder if we still need this file at all... Is it referenced by anything outside this repo?